apps/coscrad-frontend/src/components/links/links.tsx

```diff
@@ -0,0 +1,36 @@
+import LaunchIcon from '@mui/icons-material/Launch';
+import { Card, Link, Typography } from '@mui/material';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { ExternalLink } from '../../configurable-front-matter/data/configurable-content-schema';
+import { FunctionalComponent } from '../../utils/types/functional-component';
+
+const LinkPresenter = ({ title, url, description }: ExternalLink) => {
+    return (
+        <Card sx={{ p: 2, mb: 2 }}>
+            <Typography mb={0.5} variant="h3">
+                {title}
+            </Typography>
+            <Typography variant="body1" color="text.secondary">
+                {url}
+            </Typography>
+            <Typography variant="body1">{description}</Typography>
```

Comment:
Also just wondering if there's a title, people might skip putting in a description.  Should we check whether there's a description present?  Is the description required?

Reply:
I can see making the description optional. Right now, it's technically optional, in that you can just use `''` if you want it to be blank.